drivers/dac/Kconfig.mcux

```
@@ -31,3 +31,10 @@ config DAC_MCUX_DAC32_TESTOUT
 	depends on DAC_MCUX_DAC32
 	help
 	  Enable the DAC test output.
+
+config DAC_MCUX_GAU
+	bool "NXP GAU DAC driver"
+	default y
+	depends on DT_HAS_NXP_GAU_DAC_ENABLED
+	help
+	  Enable the driver for the NXP GAU D
```

Did you mean to write this?

```suggestion
	  Enable the driver for the NXP GAU DAC
```

---

ugh could have sworn I fixed this, daniel commented the same thing, thanks